share/spack/gitlab/cloud_pipelines/configs/ci.yaml

```diff
@@ -37,9 +37,10 @@ ci:
         CI_JOB_SIZE: "default"
         CI_GPG_KEY_ROOT: /mnt/key
         # SPACK_VERBOSE_SCRIPT: "1"
-      id_tokens:
-        GITLAB_OIDC_TOKEN:
-          aud: "${OIDC_TOKEN_AUDIENCE}"
```

Comment:
I don't think this block should be removed, it's how build jobs get permission to write to the appropriate bucket/prefix.